Sources/PackageGraph/RepositoryPackageContainerProvider.swift

```diff
@@ -37,8 +37,6 @@ public class RepositoryPackageContainerProvider: PackageContainerProvider {
 
     public func getContainer(for identifier: RepositorySpecifier) throws -> Container {
```

Comment:
This is useful, we need a blocking variant. Though we can create it out of async one, is that fine?